sw/spatzBenchmarks/sp-fft/main.c

```diff
@@ -113,9 +113,9 @@ int main() {
 
   // Display runtime
   if (cid == 0) {
-    // See the bottom of the file dp-fft/main.c for further info on the performance calculation
-    long unsigned int performance =
-        1000 * 5 * NFFT * log2_nfft / timer;
+    // See the bottom of the file dp-fft/main.c for further info on the
+    // performance calculation
+    long unsigned int performance = 1000 * 5 * NFFT * log2_nfft / timer;
```

Comment:
This is all just `clang-format`, right?

Reply:
yes, I run the tool with `inplace` so that during CI, when comparing against the reference style, it does not fail